Resources/public/ts/react/GridComponent.tsx

```diff
@@ -53,8 +57,9 @@ export class GridComponent extends React.Component<IProps, IState> {
         this.abortController = new AbortController();
         axios.post<GridConfiguration>(url, {
             'agGrid': {
-                'gridId': 1
-            }
+                'gridId': 1,
+                ... this.additionalAjaxRequestBody,
```

Comment:
this should be in the line below, in the main object

Reply:
Yeah you're right. As I mentioned to you. I slipped into the wrong line. I'll rectify the line.